package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@retailmenot/anchor",
-  "version": "1.1.1",
+  "version": "1.1.2",
```

Comment:
We should try to coordinate our branches so that we can do a single `npm publish` with both these changes and the `AutoComplete` fix.